sig-scalability/slos/api_call_latency.md

```diff
@@ -72,6 +72,8 @@ that all `core` components communicate with apiserver using protocol buffers.
 stale data (being served from cache) and the SLO again has to be satisfied
 independently of that. This makes the careful choice of requests in tests
 important.
+- The SLO excludes latency incurred by factors that are outside our control, specifically
+from webhooks (1.23+) and API priority & fairness queue wait times (1.27+).
```

Comment:
We need to do a bit more - meaning we need to update the SLI definition itself.

The webhooks are kind-of excluded out-of-the box in the SLO, because we're saying "default installation", which doesn't install webhooks.
But waiting time isn't.

I suggest changing the SLI definition to:

"Latency of *proceesing* ..."

And then the note you added here explains what that is (just maybe make it clear saying sth like
"By computing only the latency of processing requests, the SLI/SLO excludes..."

Reply:
Also when you change this, please update the main page too:
https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md

Reply:
I updated wording to `Latency of processing muatating / non-streaming read` and then updated the first foot note with the following:
```
<a name="footnote1">\[1\]</a> The SLI only measures latency incurred by the processing
time of the request. The processing time of a request is the moment when apiserver gets
the request to last byte of response sent to the user, excluding latency incurred by
webhooks and priority & fairness queue wait times.
```

Also updated https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md. PTAL